intl/msg_hash_es.c

```diff
@@ -1272,7 +1272,7 @@ const char *msg_hash_to_str_es(enum msg_hash_enums msg)
       case MENU_ENUM_LABEL_VALUE_CHEEVOS_DESCRIPTION:
          return "Descripción";
       case MENU_ENUM_LABEL_VALUE_CHEEVOS_HARDCORE_MODE_ENABLE:
-         return "Modo Extremo";
+         return "Achievements Modo Extremo";
```

Comment:
Achievements in spanish is Logros.
Not sure why we're using Spanglish here

And I don't think adding achievements in front of every language serves a purpose.